### DISCUSSION TITLE
How to change linkerd-viz dashboard `metricsWindow` ?

### DISCUSSION THREAD

**Discussion by taind772** [linkerd/linkerd2#9192] [Help]
I used linkerd-viz dashboard with external Prometheus. But my Prometheus `scrape_interval` was `1m`, so some metrics in the linnkerd-viz dashboard are missing. Can I use custom metricsWindow for the dashboard?

**Comment by adleong** [comment]
Hi @ndtai772!  There's been some discussion about this recently on https://github.com/linkerd/linkerd2/issues/7978 and that issue is tracking the work to add some documentation for this.

**Reply by taind772** [reply]
Actually, my problem was with the linkerd-viz dashboard, not the grafana dashboard. With Prometheus `scrape_interval` >= 1m these metrics are missing 
![image](https://user-images.githubusercontent.com/49815011/185579961-fb1072c1-ee47-4e4d-a043-2e1478aab72e.png)
